packages/ubuntu_desktop_installer/lib/pages/installation_complete/installation_complete_model.dart

```diff
@@ -0,0 +1,38 @@
+const _boldMarkdown = '**';
+const _boldMarkdownLength = _boldMarkdown.length;
+
+class TextUtils {
```

Comment:
this is simple text parsing for markdown like bold text - not sure how text from design will look like in other languages, it might be on the front, end or in the middle so this supports that text spans just need to be in markdown format

Reply:
This is not very flexible: it assumes there's exactly one bold token in the input string, so we won't be able to re-use it if in the future we want to render strings with more than one bold token, or e.g. italic script.
I would say that either we go with a more generic implementation (probably using regular expressions) or we use an existing markdown package [from pub.dev](https://pub.dev/packages?q=markdown).
If we go with the first solution (custom implementation), please extract it to a place where it can be re-used by all pages.

Reply:
My first idea was to use markdown plugin, not sure if it's worth it at current stage to import whole library for one string in the whole application? For me it's easier to import :)  will use markdown then